documentation/00_getting_started/02-hello-world.html.md

```diff
@@ -64,7 +64,7 @@ This will create a new `FlxText` instance with font size `64`, center it on the
 
 Return to your command line window - now we can compile the project. First switch to the directory containing the `Project.xml` file:
 
-```bash
+``` bash
```

Comment:
same

Reply:
it may seem so, but elsewhere in the same file, haxe is highlighted the same way ` ``` haxe`

Reply:
looks like both work, at least in github's md preview